### ISSUE TITLE
Pressing spacebar on the balloon button causes other option to look pressed

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#435]
Very likely after changes for https://github.com/phetsims/scenery/issues/888

**Comment by jessegreenberg** [comment]
> And why isn't this happening for all things that use DownUpListener like checkbox?

The listeners **are** being called in DownUpListener, but ButtonListener is what updates the state of the checkbox and its functions aren't being called from the a11y listeners.

I think these should be removed from DownUpListener. A11y button behavior has been moved to PressListener.

**Comment by jessegreenberg** [comment]
This has been fixed.
